.github/workflows/increment-plugin-versions.yml

```diff
@@ -0,0 +1,89 @@
+---
+name: Increment Plugin's Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: Log level
+        required: true
+        default: warning
+        type: choice
+        options:
+          - info
+          - warning
+          - debug
+
+jobs:
+  plugin-version-increment-sync:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        entry:
+          - {repo: alerting}
```

Comment:
Please format the file. Space after `{`, so `{ repo: alerting }`.

Reply:
I get the error from `yamllint` as 24:14 [braces] too many spaces inside braces :) 
https://github.com/opensearch-project/opensearch-build/commit/b1f75ce0d049cf18ca678f1f779f9169bc321a36